source/transactions-convenient-api/transactions-convenient-api.rst

```diff
@@ -123,11 +123,10 @@ drivers MUST enforce a 120-second timeout to limit retry behavior and safeguard
 applications from long-running (or infinite) retry loops. Drivers SHOULD use a
 monotonic clock to determine elapsed time.
 
-If an UnknownTransactionCommitResult is encountered for a commit, the commit
-operation may be retried. If the retry timeout has not been exceeded, the driver
-MUST retry a commit that fails with an error bearing the
-"UnknownTransactionCommitResult" label. If the retry timeout has been exceeded,
-drivers MUST NOT retry the commit and allow ``withTransaction`` to propagate the
+If an UnknownTransactionCommitResult error is encountered for a commit and the
+error is not MaxTimeMSExpired and the retry timeout has not been exceeded, the
+driver MUST retry the commit. If the retry timeout has been exceeded, drivers
```

Comment:
Would it make sense to replace "If the retry timeout has been exceeded" with "Otherwise,"? As-is, this paragraph doesn't technically state that drivers MUST NOT retry after a MaxTimeMSExpired error.

Reply:
Done.